examples/pytorch/pinsage/model_sparse.py

```diff
@@ -51,19 +53,19 @@ def train(dataset, args):
     device = torch.device(args.device)
 
     # Prepare torchtext dataset and vocabulary
-    fields = {}
-    examples = []
-    for key, texts in item_texts.items():
-        fields[key] = torchtext.data.Field(include_lengths=True, lower=True, batch_first=True)
+    textset = {}
+    tokenizer = get_tokenizer(None)
+
+    textlist = []
+    batch_first = True
+
     for i in range(g.number_of_nodes(item_ntype)):
```

Comment:
Generally it will be great if we can replace `number_of_nodes` with `num_nodes` as it's shorter.

Reply:
Okay